module-application/src/test/java/com/zoopi/domain/certification/service/CertificationServiceTest.java

```diff
@@ -0,0 +1,81 @@
+package com.zoopi.domain.certification.service;
+
+import static com.zoopi.domain.certification.entity.BloodDonationType.*;
+import static com.zoopi.util.FunctionalUtils.*;
+import static org.mockito.Mockito.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import org.junit.jupiter.api.Test;
+
+import com.zoopi.domain.certification.dto.CertDetailDto;
+import com.zoopi.domain.certification.entity.BloodDonationDetail;
+import com.zoopi.domain.certification.entity.BloodDonationHistory;
+import com.zoopi.domain.certification.repository.BloodDonationDetailRepository;
+import com.zoopi.domain.certification.repository.BloodDonationHistoryRepository;
+import com.zoopi.domain.chat.entity.ChatMessage;
+import com.zoopi.domain.chat.entity.ChatRoom;
+import com.zoopi.domain.chat.entity.ChatRoomStatus;
+import com.zoopi.domain.chat.entity.MessageType;
+import com.zoopi.domain.chat.repository.ChatMessageRepository;
+import com.zoopi.domain.hospital.entity.Hospital;
+import com.zoopi.domain.pet.entity.Pet;
+
+class CertificationServiceTest {
+
+	private final BloodDonationHistoryRepository historyRepository = mock(BloodDonationHistoryRepository.class);
+	private final BloodDonationDetailRepository detailRepository = mock(BloodDonationDetailRepository.class);
+	private final ChatMessageRepository chatMessageRepository = mock(ChatMessageRepository.class);
+
+	private final CertificationService certificationService = new CertificationService(
+		historyRepository,
+		detailRepository,
+		chatMessageRepository
+	);
```

Comment:
```suggestion
@ExtendWith(MockitoExtension.class)
class CertificationServiceTest {

        @Mock
	private final BloodDonationHistoryRepository historyRepository;
        @Mock
	private final BloodDonationDetailRepository detailRepository;
        @Mock
	private final ChatMessageRepository chatMessageRepository;
        @InjectMocks
	private final CertificationService certificationService;
```

이렇게 작성하면 코드가 더 깔끔해지는 것 같습니다!

Reply:
테스트코드 관련해서는 작성해주신 방법 좋은것 같아요! 코틀린에서 사용하던 방법을 그대로 가져오다보니 조금 비효율적이 된것 같네요! 수정해보도록하겠습니다! 💯 💯